### ISSUE TITLE
 Improve keyboard shortcut focus when renaming an asset

### ISSUE THREAD

**Issue by jaygeorge** [statamic/a11y#10]
## Description of the Problem

When renaming an asset in the CP the keyboard focus does not shift to the new modal, instead, it stays in the background

I think there are a few other instances where the modal implementation behaves this way too such as deleting an asset and the lack of keyboard focus on that modal.

## How to Reproduce

1. Go to 'Assets' in the CP
2. Select an asset and click the 'rename' button
3. Press the tab key on your keyboard and observe how the keyboard focus is stuck behind the modal

## What is the expected result?

As soon as you click 'rename to rename an asset the keyboard focus should shift to the modal, highlighting the 'filename' input so you can just start typing to rename.

## What happens instead?

Hitting tab starts focusing elements behind the modal. It is possible to eventually get to the modal content but it takes a lot of keystrokes
